src/components/PaymentLog.vue

```diff
@@ -1171,12 +1291,13 @@ export default {
         //     icon: "error",
         //     title: "Plan is not available"
         // });
-        this.test1 = true
-        this.repaymentCircle = '0'
-        this.rDuration = '0'
-        this.fPayment = '0'
-        this.rPayment = '0'
-        this.pPrice = '0'
+       this.test1 = true
```

Comment:
What is test1?

Reply:
that is for the checks before submit button is enabled

Reply:
we should be using a better descriptive name

Reply:
done